deltachat-ios/Helper/ImageFormat.swift

```diff
@@ -67,4 +68,35 @@ extension ImageFormat {
         return loadImageFrom(data: imageData)
     }
 
+    public static func saveImage(image: UIImage) -> String? {
+        if image.sd_isAnimated,
+           let data = image.sd_imageData() {
```

Comment:
does that result in a recoding?

that may be a bit unfortunately wrt stickers (as it would make showing every received sticker in a selector harder as they the same stickers may have different binaries)

Reply:
answering myself: for animated images, there is "No more extra encoding process." says comment atop of sd_imageData.

however, for not-animated stickers, recoding is still always done (that are handled below).